vpr/src/analytical_place/read_atom_netlist.cpp

```diff
@@ -0,0 +1,175 @@
+/**
+ * @file
+ * @author  Alex Singer
+ * @date    September 2024
+ * @brief   Definition of the read_atom_netlist method, used for generating an
+ *          APNetlist from the results of the Prepacker.
+ */
+
+#include "read_atom_netlist.h"
+#include "ap_netlist.h"
+#include "atom_netlist.h"
+#include "atom_netlist_fwd.h"
+#include "netlist_fwd.h"
+#include "partition.h"
+#include "partition_region.h"
+#include "prepack.h"
+#include "region.h"
+#include "user_place_constraints.h"
+#include "vpr_types.h"
+#include "vtr_assert.h"
+#include "vtr_geometry.h"
+#include "vtr_time.h"
+#include <unordered_set>
+#include <vector>
+
```

Comment:
read_atom_netlist is a bit of a confusing name. We normally use read_* for file input, and we do read a blif netlist already.  I think this is gen_APNetlist_from_atoms () or something like that or create_APNetlist_from_atom_netlist ()

Reply:
Very good point. This was historical. Originally it read the atom netlist directly. I have since made it take the prepacked solution. I like the name `gen_ap_netlist_from_atoms`.